grudge/array_context.py

```diff
@@ -225,6 +225,11 @@ class MPIPytatoArrayContextBase(MPIBasedArrayContext):
     def __init__(
             self, mpi_communicator, queue, *, mpi_base_tag, allocator=None
             ) -> None:
+
+        if allocator is None:
+            from warnings import warn
+            warn("No memory allocator specified, please pass one.")
+
```

Comment:
Thanks, I wonder if it might be better to put this in PyOpenCLArrayContext and PytatoPyOpenCLArrayContext.

Reply:
Done in [875de83](https://github.com/inducer/grudge/pull/240/commits/875de839c206b3b5e4690c391ed0eadb28be0adb)